types/ckeditor__ckeditor5-ui/src/colorgrid/colorgrid.d.ts

```diff
@@ -15,7 +15,7 @@ export default class ColorGridView extends View {
 export interface ColorDefinition {
     color: string;
     label: string;
-    options: {
-        hasBorder: boolean;
+    options?: {
+        hasBorder?: boolean;
```

Comment:
why this change?
(sorry, I'm not a vivid user of the CKEditor), just comparing to:
https://github.com/ckeditor/ckeditor5/blob/824d72c51a7672f57cbddf31634182925f671332/packages/ckeditor5-ui/src/colorgrid/colorgridview.js#L179-L213

Reply:
Reverted. My bad.